### DISCUSSION TITLE
Pause and Edit

### DISCUSSION THREAD

**Discussion by Boladin** [nozwock/yanu#22] [Ideas]
It would be great if there was a break in between. So you can edit file or replace in ROM. Thank you very much as I managed to update ROM on Android by your method.

**Comment by nozwock** [comment]
Literally
```
yanu-cli unpack --help
# And
yanu-cli repack --help
```

**Reply by nozwock** [reply]
Modding/patching experience will be improved in future versions.

**Reply by Boladin** [reply]
Muito obrigado, pensei que era gringo, já Br melhor ainda conversa. Muito obrigado

**Reply by Boladin** [reply]
> Literalmente
> 
> ```
> yanu-cli unpack --help
> # And
> yanu-cli repack --help
> ```

Isso serve pra que, só ruim no inglês e programação.

**Reply by nozwock** [reply]
There's a new gui in the works right now #23

**Comment by nozwock** [comment]
[0.10.0](https://github.com/nozwock/yanu/releases/tag/0.10.0) is released with the new GUI, you can unpack and pack from there which is effectively what you're suggesting.

**Reply by Boladin** [reply]
Valeu já consegui, sei que isso não devia perguntar, mais ele tem suporte até qual firmware? Pois o Zelda tiktok não consegui fazer update. Trava no processo

**Reply by nozwock** [reply]
On pc, I heard people say that it might take some hours for Totk. It gets stuck on "Verifying directories" or something for quite some time.
